dev/setup-k3ds

```diff
@@ -7,7 +7,7 @@ docker_mirror=${docker_mirror-}
 
 if [ -n "$docker_mirror" ]; then
   TMP_CONFIG="$(mktemp)"
-  trap "rm -f $TMP_CONFIG" EXIT
+  trap 'rm -f $TMP_CONFIG' EXIT
```

Comment:
I don't see how that matters. It's always the TMP_CONFIG from the previous line, we don't want to evaluate it later on exit.